docs/howto/custom-operator.rst

```diff
@@ -85,7 +94,7 @@ Let's extend our previous example to fetch name from MySQL:
                     *args, **kwargs) -> None:
                 super().__init__(*args, **kwargs)
                 self.name = name
-                self.conn_id = conn_id
+                self.mysql_conn_id = mysql_conn_id
```

Comment:
This is a better name, but the constructor arg is called `conn_id` still and needs updating.

Reply:
updated now